tests/flytekit/unit/core/test_type_engine.py

```diff
@@ -1710,3 +1710,7 @@ def test_is_annotated(t, expected):
 )
 def test_get_underlying_type(t, expected):
     assert get_underlying_type(t) == expected
+
+
+def test_dict_get():
+    assert DictTransformer.get_dict_types(None) == (None, None)
```

Comment:
why are we adding this?

Reply:
felt like it was missing, just wanted to be sure it could handle None.